### DISCUSSION TITLE
Création du site statique du programme MeC

### DISCUSSION THREAD

**Discussion by houssamkhalifeh** [VilledeMontreal/MontrealEnCommun#6] [Ideas]
A partir de la documentation ajoutée par les contiributeurs, créer le site du programme MeC.

**Comment by vince-fabmob** [comment]
Test ~~Vincent~~ Patrick

**Comment by vince-fabmob** [comment]
Bonjour Houssam, est-ce que la structure permet aux partenaires d'ajouter des documents (PDF, word, etc. )? Sinon, je crois que tout est ok pour aller de l'avant avec les répertoires.
